src/rgw/rgw_log.cc

```diff
@@ -128,14 +128,13 @@ class UsageLogger {
     round_timestamp = ts.round_to_hour();
   }
 
-  void insert_user(utime_t& timestamp, const rgw_user& user, rgw_usage_log_entry& entry) {
+  void insert_user(utime_t& timestamp, const string& user, rgw_usage_log_entry& entry) {
```

Comment:
It seems 
[tenant is record in rgw_bucket](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_common.h#L930) ,  [rgw_usage_log_entry is name with both user and bucket](https://github.com/ceph/ceph/blob/master/src/cls/rgw/cls_rgw_types.h#L798), maybe `user_name:subuser_name` would not conflict with multi-tenant bucket?

But `user_name:subuser_name` really has conflict with following condition:

```
$ radosgw-admin user create --uid=user:subuser  --display-name="User"
{
    "user_id": "user:subuser",   <============= `user_name:subuser_name`
    "subusers": [],
    "keys": [
        {
            "user": "user:subuser",
            "access_key": "5S5DXB3J42HJ4UOOJWFN",
            "secret_key": "3fSNSbdYBpDgt8K8eEQVeADBijh30d7eFFFRUZCQ"
        }
    ],
}

$ radosgw-admin subuser create --key-type=s3 --uid=user --subuser=subuser \
--access=full --access-key=access  --secret-key=secret
{
    "user_id": "user",
    "subusers": [
        {
            "id": "user:subuser", <============= `user_name:subuser_name`
            "permissions": "full-control"
        }
    ],
    "keys": [
        {
            "user": "user",
            "access_key": "NCKNOPY2J4RSD605IPVZ",
            "secret_key": "nQUZJGededkFW51uzl7PgATG1ox2t7PZj42HyBrz"
        },
        {
            "user": "user:subuser",
            "access_key": "access",
            "secret_key": "secret"
        }
    ],
}

```

Maybe we can treat this as a operation convention, `do not create user that uid contains : ` . Or we can add some user naming checking? Current `OPT_USER_CREATE` in `radosgw-admin` did not has such checking.